eyewear_shop/demo/website_view.xml

```diff
@@ -0,0 +1,1657 @@
+                        <span t-field="website.logo" t-options="{'widget': 'image', 'style': 'max-width: 200px'}" role="img" t-att-aria-label="'Logo of %s' % website.name" t-att-title="website.name"/>
+                    </a>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="ir_ui_view_2636" model="ir.ui.view">
+        <field name="name">option_header_off_canvas_template_header_hamburger</field>
+        <field name="inherit_id" ref="website.template_header_hamburger"/>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="key">website.option_header_off_canvas_template_header_hamburger</field>
+        <field name="mode">extension</field>
+        <field name="priority">32</field>
+        <field name="type">qweb</field>
+        <field name="arch" type="xml">
+            <data inherit_id="website.template_header_hamburger" active="False" priority="32">
+                <xpath expr="//div[@id='oe_structure_header_hamburger_2']" position="replace"/>
+                <xpath expr="//div[@id='oe_structure_header_hamburger_3']" position="replace"/>
+            </data>
+        </field>
+    </record>
+    <record id="ir_ui_view_2637" model="ir.ui.view">
+        <field name="name">option_header_off_canvas_template_header_sidebar</field>
+        <field name="inherit_id" ref="website.template_header_sidebar"/>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="key">website.option_header_off_canvas_template_header_sidebar</field>
+        <field name="mode">extension</field>
+        <field name="priority">32</field>
+        <field name="type">qweb</field>
+        <field name="arch" type="xml">
+            <data inherit_id="website.template_header_sidebar" active="False" priority="32">
+                <xpath expr="//div[@id='oe_structure_header_sidebar_1']" position="attributes">
+                    <attribute name="class" add="d-none d-lg-block" separator=" "/>
+                </xpath>
+            </data>
+        </field>
+    </record>
+    <record id="ir_ui_view_2639" model="ir.ui.view">
+        <field name="name">List View (by default)</field>
+        <field name="inherit_id" ref="website_sale.products"/>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="key">website_sale.products_list_view</field>
+        <field name="mode">extension</field>
+        <field name="type">qweb</field>
+        <field name="arch" type="xml">
+            <data inherit_id="website_sale.products" active="False" name="List View (by default)">
+                <xpath expr="//div[@id='products_grid']" position="after">
+                    <!-- Nothing to do, this view is only meant to allow the server -->
+                    <!-- to know if the list view layout should be used -->
+                </xpath>
+            </data>
+        </field>
+    </record>
```

Comment:
Do we really need this?
Try to remove if not causing any error

Reply:
It is required because it is responsible for showing list view when we open shop from website